p2p/src/main/kotlin/maru/p2p/discovery/MaruDiscoveryService.kt

```diff
@@ -45,7 +46,7 @@ class MaruDiscoveryService(
   }
 
   companion object {
-    const val FORK_ID_HASH_FIELD_NAME = "mfidh"
+    const val FORK_ID_HASH_FIELD_NAME = "eth2"
```

Comment:
Why are we changing it to `eth2`?

Reply:
I was thinking about this. Now I'm not sure about this anymore. eth2 is the field that they are using in the CLs. I thought that when we are using eth2 we could find out when there are a lot of CLs trying to connect. I think I will change that back ...